File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        if(x<0){
            return false;
        }
        long long cont =0;
        long long temp = x;
        while(temp!=0){
            int digit = temp % 10;
            cont = cont *10 +digit;
            temp = temp /10;

        }
        return (cont== x);
        
    }
};

File: 0009-palindrome-number/0009-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(s) {
    let temp = 0;
    let x = Math.abs(s)
    while(x>0){
        let d = x%10;
        temp = temp*10 + d
        x = Math.floor(x/10);
    }
    if(temp == s) return true
    return false
};